### PR TITLE
Add Pages

### PR DESCRIPTION
**Pull request by SebastianStupp**
Add Pages

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,14 +11,14 @@
 
         <footer>
             <div class="footer-nav">
-                <a href="#">
-                    <botton>🏂</botton>
+                <a href="index.html">
+                    <botton class="btn btnactive">🏂</botton>
```
Comment (lmachens) [inline index.html]: Typo:
It should be `button` instead of `botton`.
This happens multiple times

File: second.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+        <footer>
+            <div class="footer-nav">
+                <a href="index.html">
+                    <botton class="btn">🏂</botton>
+                </a>
+                <a href="second.html">
+                    <botton class=" btn btnactive">🪀</botton>
+                </a>
+                <a href="thrid.html">
```
Comment (lmachens) [inline second.html]: Typo: `third` instead of `thrid`

File: index.html (hunk 1 above)
Comment (SebastianStupp) [reply index.html]: Don't drink and drive.. seems to be true 🍺 thx :)

File: second.html (hunk 2 above)
Comment (SebastianStupp) [reply second.html]: thx :)

**Pull request merged**